lib/katello/tasks/pulp3_content_switchover.rake

```diff
@@ -0,0 +1,19 @@
+require File.expand_path("../engine", File.dirname(__FILE__))
+
+namespace :katello do
+  desc "Runs a Pulp 3 migration of pulp3 hrefs to pulp ids for supported content types."
+  task :pulp3_content_switchover => :environment do
+    content_types = Katello::Pulp3::Migration.content_types_for_migration
+
+    content_types.each do |content_type|
+      if content_type.model_class.where(migrated_pulp3_href: nil).any?
+        $stderr.print("ERROR: at least one #{content_type.model_class.table_name} record has migrated_pulp3_href NULL value")
+        exit 1
```

Comment:
would it make sense to just raise an exception?

Reply:
Previous rake tasks seemed to have used SystemExit. I don't have a strong opinion either way.